### PR TITLE
Feature/collections length

### PR DESCRIPTION
**Pull request by wendidy**
[dockstore/4074](https://github.com/dockstore/dockstore/issues/4074)
Passed the added integration test locally. Wondering what the failed tests are here in ` io.dockstore.client.cli.CromwellIT`, it seems like this happened after merging develop in?

### CODE REVIEW

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java [hunk 1]
```diff
@@ -103,7 +103,10 @@ public OrganizationResource(SessionFactory sessionFactory) {
     @Operation(operationId = "getApprovedOrganizations", summary = "List all available organizations.", description = "List all organizations that have been approved by a curator or admin, sorted by number of stars.")
     public List<Organization> getApprovedOrganizations() {
         List<Organization> organizations = organizationDAO.findApprovedSortedByStar();
-        organizations.stream().forEach(org -> Hibernate.initialize(org.getAliases()));
+        organizations.stream().forEach(org -> {
+            Hibernate.initialize(org.getAliases());
+            org.setCollectionsLength(org.getCollections().size());
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java:108]: How many additional SQL queries does this cause? If it's 0, then it's fine, if it's the number of collections, then we might want to think about a dedicated query.

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java [hunk 2]
```diff
@@ -103,7 +103,10 @@ public OrganizationResource(SessionFactory sessionFactory) {
     @Operation(operationId = "getApprovedOrganizations", summary = "List all available organizations.", description = "List all organizations that have been approved by a curator or admin, sorted by number of stars.")
     public List<Organization> getApprovedOrganizations() {
         List<Organization> organizations = organizationDAO.findApprovedSortedByStar();
-        organizations.stream().forEach(org -> Hibernate.initialize(org.getAliases()));
+        organizations.stream().forEach(org -> {
+            Hibernate.initialize(org.getAliases());
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java:107]: Why does this initialise aliases?

**Review by denis-yuen — COMMENTED** [review]
Can ignore https://ucsc-cgl.atlassian.net/browse/SEAB-2622

**Review by coverbeck — APPROVED** [review]
Discussed with Wendy over Slack, and it's one extra query per org, which I think is acceptable, although if orgs become very numerous we might want to revisit.

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java (hunk 1 above)
Comment (wendidy) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java:108]: Discussed with Charles on Slack. There's one SQL statement per organization, so will leave it as it is for now, might cause issues when the number of organization grows

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java (hunk 2 above)
Comment (wendidy) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java:107]: @denis-yuen It was there before, should I make some modifications?

Comment (denis-yuen) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java:107]: Oh nvm, just got reformatted.

**Comment by garyluu** [comment]
Merge from develop, cromwellIT problematic tests were removed. also, yq issue

**Review by denis-yuen — COMMENTED** [review]
Re-noting that the openapi3 yaml seems to have a lot of un-necessary change, you might be dealing with an incorrect yq version

**Review by denis-yuen — APPROVED** [review]
with current openapi.yaml changes (roughly 3 lines added, 1 deleted)

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**